### PR TITLE
fix(InfinityScroller): forward load button props

### PR DESCRIPTION
**Pull request by joakbjerk**
- [x] Centralize load button props
- [x] Allow for custom button with load button props

### CODE REVIEW

File: packages/dnb-eufemia/src/components/pagination/Pagination.d.ts [hunk 1]
```diff
@@ -146,6 +146,10 @@ export interface PaginationProps
    * Used during infinity mode. If `use_load_button` is set to `true`, then a button is show on the bottom. If the `startup_page` is higher than 1. Defaults to `Vis mer innhold`.
    */
   load_button_text?: string;
+  /**
+   * Used during infinity mode. Sets the icon position on the `use_load_button`. Default: `left`.
+   */
+  load_button_icon_position?: string;
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/pagination/Pagination.d.ts:152]:
```suggestion
  loadButtonIconPosition?: ButtonIconPosition;
```

Needs to be imported.

File: packages/dnb-eufemia/src/components/pagination/Pagination.d.ts [hunk 2]
```diff
@@ -327,7 +331,10 @@ interface PaginationInstanceProps extends SpacingProps {
    * Used during infinity mode. If `use_load_button` is set to true, then a button is show on the bottom. If the `startup_page` is higher than 1. Defaults to `Vis mer innhold`.
    */
   load_button_text?: string;
-  className?: string;
+  /**
+   * Used during infinity mode. Sets the icon position on the `use_load_button`. Default: `left`.
+   */
+  load_button_icon_position?: string;
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/pagination/Pagination.d.ts:337]:
```suggestion
  loadButtonIconPosition?: ButtonIconPosition;
```

File: packages/dnb-eufemia/src/components/pagination/Pagination.d.ts [hunk 3]
```diff
@@ -502,6 +509,10 @@ interface InfinityMarkerProps extends SpacingProps {
    * Used during infinity mode. If `use_load_button` is set to true, then a button is show on the bottom. If the `startup_page` is higher than 1. Defaults to `Vis mer innhold`.
    */
   load_button_text?: string;
+  /**
+   * Used during infinity mode. Sets the icon position on the `use_load_button`. Default: `left`.
+   */
+  load_button_icon_position?: string;
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/pagination/Pagination.d.ts:515]:
```suggestion
  loadButtonIconPosition?: ButtonIconPosition;
```

File: packages/dnb-eufemia/src/components/pagination/Pagination.js [hunk 4]
```diff
@@ -92,6 +92,7 @@ const paginationPropTypes = {
   more_pages: PropTypes.string,
   is_loading_text: PropTypes.string,
   load_button_text: PropTypes.string,
+  load_button_icon_position: PropTypes.string,
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/pagination/Pagination.js:95]:
```suggestion
  loadButtonIconPosition: PropTypes.string,
```

File: packages/dnb-eufemia/src/components/pagination/Pagination.js [hunk 5]
```diff
@@ -125,6 +126,7 @@ const paginationDefaultProps = {
   more_pages: null,
   is_loading_text: null,
   load_button_text: null,
+  load_button_icon_position: 'left',
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/pagination/Pagination.js:129]:
```suggestion
  loadButtonIconPosition: 'left',
```

**Review by tujoworker — COMMENTED** [review]
I suggest to add a rather a property called `loadButtonProps` to allow all kinds of props. Or even just `loadButton`, and allow a "custom" component to be served.

**Comment by joakbjerk** [comment]
> I suggest to add a rather a property called `loadButtonProps` to allow all kinds of props. Or even just `loadButton`, and allow a "custom" component to be served.

Sounds good, I agree that a loadButtonProps is a better solution, so we can centralize all the load button specific props at one place 👍 The `fallback_element` prop allows for a custom loadButton, but maybe we should rename this one, or swap it out for the new loadButton prop at one point?

**Review by tujoworker — APPROVED** [review]
Nice 👏

**Pull request merged**